app-modules/service-management/database/seeders/ChangeRequestTypeSeeder.php

```diff
@@ -51,15 +50,11 @@ public function run(): void
         ];
 
         foreach ($types as $type) {
-            $changeRequestType = ChangeRequestType::factory()
+            ChangeRequestType::factory()
                 ->create([
                     'name' => $type[0],
                     'number_of_required_approvals' => $type[1],
                 ]);
-
-            if (app()->environment('local', 'testing')) {
-                $changeRequestType->userApprovers()->attach(User::where('email', config('local_development.super_admin.email'))->first()->id);
-            }
```

Comment:
Do we need to get rid of this chunk? It's useful for local.

Reply:
For now yes, as we won't have a user when this runs by default. So we need a better way to handle this. I'll creat a ticket for it